### PR TITLE
added perl as a build dependency for parallel

### PR DESCRIPTION
**Pull request by JensTimmerman**

### CODE REVIEW

**Comment by JensTimmerman** [comment]
ok, so the tests fail because there are no perl's for the toolchains I added, however, I tested this, but this does actually breaks the build, I'm running into this issue: 
`https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=724123`
for some reason this does not occur with RHEL's perl, but it does on CENTOS and debian, so I actually consider the old parallel .eb's broken...

**Comment by JensTimmerman** [comment]
Just tested this out, parallel build for version `20130122`  works with Perl-5.16.3, but fails with  Perl-5.20.1, so changing that.

**Comment by JensTimmerman** [comment]
ok, I changed to an older perl where we had one, not sure what to do for old toolchains where we have no perl yet, do you want me to put time in adding perl to an old toolchain we're never going to use?

Also, what to do for the GCC one? Add a perl for it aswell? or remove it?

**Comment by boegel** [comment]
add the missing easyconfigs for Perl, it's `-bare` anyway?

File: easybuild/easyconfigs/p/parallel/parallel-20141122-GCC-4.9.2.eb [hunk 1]
```diff
@@ -11,6 +11,11 @@ source_urls = ['http://ftp.gnu.org/gnu/parallel']
 
 toolchain = {'name': 'GCC', 'version': '4.9.2'}
 
+# warning, parallel is a perl script and should depend on perl
```
Comment (boegel) [inline easybuild/easyconfigs/p/parallel/parallel-20141122-GCC-4.9.2.eb]: drop the rather silly warning, add a Perl easyconfig for GCC toolchain

**Comment by JensTimmerman** [comment]
I can look into adding the perls if I find some time later this year

**Comment by boegel** [comment]
@JensTimmerman: adding those easyconfig is pretty trivial, no?

just add them, and we can fire a test report to see if things work out

**Pull request merged**